src/ICSharpCode.SharpZipLib/Zip/ZipFile.cs

```diff
@@ -571,9 +571,9 @@ internal ZipFile()
 		/// <exception cref="System.IO.IOException">
 		/// An i/o error occurs.
 		/// </exception>
-		public void Close()
+		public void Dispose()
```

Comment:
File should have close available still

Reply:
It's defined at line [#3267](https://github.com/icsharpcode/SharpZipLib/pull/407/files/c41848719d3ecf89c78688eb0d48298861a68e2f#diff-90f6fd221702a6a05926e8890e900c78R3267)